test-integration.sh

```diff
@@ -35,7 +36,7 @@ then
 fi
 
 # compile the integration test binary
-go test -test.c -test.tags=${TAGS} -test.covermode ${COVERMODE} \
+go test ${TEST_OPTS} -test.c -test.tags=${TAGS} -test.covermode ${COVERMODE} \
```

Comment:
Can you please use the same manner of constructing tags used in `process-cover-package.sh`? https://github.com/m3db/ci-scripts/blob/ec7ca2ce8dfbf89017ea9f7bff7be819e2279eb1/process-cover-package.sh#L10-L15

Passing opts as a string like this is more error-prone to bad args. I know we don't have shellcheck in this repo so there's no way for linting to catch.

Reply:
that pattern doesn't really work for this. instead i'll just make an explicit `RACE=true` var and then use it to build up the TEST_OPTS internally. a similar pattern can be used in the future for more opts